openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KurentoHandler.java

```diff
@@ -346,9 +349,19 @@ JSONArray getTurnServers(final boolean test) {
 					turn.put("username", turnUser)
 						.put("credential", turnSecret);
 				}
-				final String fturnUrl = "turn:" + turnUrl;
-				turn.put("url", fturnUrl); // old-school
-				turn.put("urls", fturnUrl);
+
+				JSONArray urls = new JSONArray();
+				final String[] turnUrls = turnUrl.split(",");
+				for (int i = 0; i < turnUrls.length; i++) {
+					final String url = turnUrls[i];
+					if (!url.startsWith("stun:") && !url.startsWith("stuns:") && !url.startsWith("turn:") && !url.startsWith("turns:")) {
+						urls.put("turn:" + url);
+					} else {
+						urls.put(url);
+					}
+				}
+				turn.put("urls", urls);
```

Comment:
Are you sure it is safe to drop `turn.put("url", fturnUrl); // old-school` ?

Reply:
It was removed from the specification in June 2013 and [its use is discouraged](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer/url). But I can bring it back and populate with first url.

Reply:
No need to add it back in this case :)